shared/scss/_vars.scss

```diff
@@ -1,6 +1,6 @@
 $font__face--default: system, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
     'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
-$font__face--browser-alt: 'DDG_ProximaNova';
```

Comment:
@shakyShane I added this as a fallback, as unless the popup is embedded in the extension, the `DDG_ProximaNova` will 404. This makes the popup look nicer for those causes (primarily local testing).